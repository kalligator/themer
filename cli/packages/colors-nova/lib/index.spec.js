const { colors } = require('./index');

describe('colors-nova', () => {
  it('should export valid themer color set', () => {
    expect(colors).toMatchSnapshot();
  });
});
