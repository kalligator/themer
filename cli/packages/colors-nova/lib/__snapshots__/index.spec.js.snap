// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colors-nova should export valid themer color set 1`] = `
Object {
  "dark": Object {
    "accent0": "#DF8C8C",
    "accent1": "#F2C38F",
    "accent2": "#DADA93",
    "accent3": "#A8CE93",
    "accent4": "#7FC1CA",
    "accent5": "#83AFE5",
    "accent6": "#9A93E1",
    "accent7": "#D18EC2",
    "shade0": "#3C4C55",
    "shade1": "#495A64",
    "shade2": "#556873",
    "shade3": "#6A7D89",
    "shade4": "#899BA6",
    "shade5": "#A7B8C2",
    "shade6": "#C5D4DD",
    "shade7": "#E6EEF3",
  },
}
`;
